### PR TITLE
Add migration file for Django>=1.7

### PR DESCRIPTION
**Pull request by ro5k0**

### CODE REVIEW

**Comment by rosco77** [comment]
Any chance of getting this added please?

**Comment by juliomalegria** [comment]
Is there a way this could break/affect users using Django<1.7?

**Comment by rosco77** [comment]
Not that I'm aware of.  However, since 1.7 is the longest version that is no supported officially I don't think that's an issue personally.  In my mind it would be much better to support people using 1.7 onwards by having the migration files in there so that people don't have to try and debug an error when using this great package.

**Comment by flavioamieiro** [comment]
+1, please include migrations so we can use this package with newer versions of django :)

**Comment by juliomalegria** [comment]
Sorry I've taken forever. Would you mind regenerating the migration for it to be up to date with the latest changes?

**Comment by hackdna** [comment]
Would it be possible to merge this soon and publish a new release? This would help us (and others) avoid the hassle of having a [local migration](https://github.com/refinery-platform/refinery-platform/commit/740531eb77cf152b2c6a35044d11cf3906d76412#diff-6f2564ba645b9b88929a9d507b9a3b7a) for this app.

**Comment by juliomalegria** [comment]
Hey @ro5k0, @hackdna 

The main problem with this is that the `ChunkedUpload` can be an abstract model (controlled by `CHUNKED_UPLOAD_ABSTRACT_MODEL ` in the settings). If that's the case, I don't know how will Django behave with the migration.

I see in the migration file there is `options={'abstrct': False}`. Maybe we should import `ABSTRACT_MODEL` from the settings to control that option?

```python
# chunked_upload/migrations/0001_initial.py
from .settings import ABSTRACT_MODEL
# ...

class Migration(migrations.Migration):
  # ...
            options={
                'abstract': ABSTRACT_MODEL,
            },
```

File: chunked_upload/migrations/0001_initial.py [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='ChunkedUpload',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('upload_id', models.CharField(default=chunked_upload.models.generate_upload_id, editable=False, max_length=32, unique=True)),
```
Comment (juliomalegria) [inline chunked_upload/migrations/0001_initial.py]: This should be `default=chunked_upload.settings.UPLOAD_TO`

**Comment by shaw2thefloor** [comment]
Would really like a fix to this issue :)

**Comment by hackdna** [comment]
It looks like using `settings.CHUNKED_UPLOAD_ABSTRACT_MODEL` should work. There is already `settings.AUTH_USER_MODEL`.

**Comment by jerinpetergeorge** [comment]
Thi change was done in #53

**Pull request closed without merging**